Daily test failed on Wed Apr 15 2026

Run listed here: https://github.com/AlexWaygood/typeshed-stats/actions/runs/24433728505

I think a new release of packaging broke this:

```
    def test_stub_version_repr() -> None:
        version = StubVersion("==1.2.*")
        assert repr(version) == "StubVersion('==1.2.*')"
>       assert SpecifierSet.__repr__(version) == "<SpecifierSet('==1.2.*')>"
E       assert "<StubVersion('==1.2.*')>" == "<SpecifierSet('==1.2.*')>"
E         
E         - <SpecifierSet('==1.2.*')>
E         + <StubVersion('==1.2.*')>
```

That test seems a bit dubious anyway though, not sure why it's written that way.

--- a/tests/test_gather.py
+++ b/tests/test_gather.py
@@ -19,7 +19,6 @@ from unittest import mock
 # Some tests assert behaviour that's predicated on rich not yet being imported
 import aiohttp
 import pytest
-from packaging.specifiers import SpecifierSet
 from packaging.version import Version
 from pytest_mock import MockerFixture
 from pytest_subtests import SubTests

--- a/tests/test_gather.py
+++ b/tests/test_gather.py
@@ -1283,4 +1283,3 @@ def test_exceptions_bubble_up(typeshed: Path) -> None:
 def test_stub_version_repr() -> None:
     version = StubVersion("==1.2.*")
     assert repr(version) == "StubVersion('==1.2.*')"
-    assert SpecifierSet.__repr__(version) == "<SpecifierSet('==1.2.*')>"

